controller/backing_image_controller.go

```diff
@@ -212,12 +212,12 @@ func (bic *BackingImageController) syncBackingImage(key string) (err error) {
 			log.Info("Need to wait for all replicas stopping using this backing image before removing the finalizer")
 			return nil
 		}
-		cleanUpTmpFileFinished, err := bic.IsBackingImageDataSourceCleanup(backingImage)
+		cleaned, err := bic.IsBackingImageDataSourceCleaned(backingImage)
 		if err != nil {
 			return err
 		}
-		if !cleanUpTmpFileFinished {
-			log.Info("Requesting to clean up Backing Image Data Source first before cleaning up Backing Image Manager")
+		if !cleaned {
+			log.Info("Need to wait until backing image data source is cleaned before removing the finalizer")
```

Comment:
Is this reachable? There is already an error when `IsBackingImageDataSourceCleaned` returns `false`

Reply:
yes you are right, 
`cleaned=false` will always follow an error returned
so it will be caught already